.gitignore

```diff
@@ -14,3 +14,4 @@ src/c/test/smoketest.o
 src/c/test/smoketest2
 src/c/test/smoketest2.o
 src/c/weaver/build/
+src/python/test/test-results/
```

Comment:
This should be a commit of its own and an explantion.

Reply:
I tried editting the commit itself but I cant remove the files added to the commit. So I changed the commit log to specify all three changes. if that is not good, I will have to create a new pull request.

Reply:
You can modify individual commit messages and add/remove files:
`git rebase -i HEAD~n`, where `n` is the last n commits

This will put you on that commit, and you'll be able to `git add` or `git rm` files and you edit the message with `git commit --amend`. Afterwards, run `git rebase --continue`.

Reply:
Thanks that worked.

Reply:
One other small thing: In the commit messages, can you split the title and the body by adding an empty line between the two? Title is usually 80 chars to give a high level view of what the commit is.

Reply:
Added

Reply:
@hariharan-devarajan Can you adjust the width of the body of your commit messages to also be 80 chars?